ledger/applications_test.go

```diff
@@ -1295,3 +1299,19 @@ func testAppAccountDeltaIndicesCompatibility(t *testing.T, source string, accoun
 	a.Contains(blk.Payset[0].ApplyData.EvalDelta.LocalDeltas[accountIdx], "lk1")
 	a.Equal(blk.Payset[0].ApplyData.EvalDelta.LocalDeltas[accountIdx]["lk1"].Bytes, "local1")
 }
+
+func TestLogicLedgerSetLog(t *testing.T) {
+	a := require.New(t)
+
+	addr := getRandomAddress(a)
+	aidx := basics.AppIndex(1)
+	c := newCowMock([]modsData{
+		{addr, basics.CreatableIndex(aidx), basics.AppCreatable},
+	})
+	l, err := newLogicLedger(c, aidx)
+	a.NoError(err)
+	a.NotNil(l)
+	tv := basics.TealValue{Type: basics.TealBytesType, Uint: 1}
```

Comment:
I think this is a mistake? It should probably be `basics.TealValue{Type: basics.TealBytesType, Bytes: someByteString}` -- this is another argument for why `AppendLog` might be better off taking a byte slice as an argument.